client/lib/reader-teams/test/team-store-test.js

```diff
@@ -1,16 +1,17 @@
-require( 'lib/react-test-env-setup' )();
-
 // External dependencies
-var expect = require( 'chai' ).expect,
+const expect = require( 'chai' ).expect,
 	Dispatcher = require( 'dispatcher' ),
 	sinon = require( 'sinon' );
 
 // Internal dependencies
-var TeamStore = require( '../index' ),
-	ActionTypes = require( '../constants' ).action;
+const TeamStore = require( '../index' ),
+	ActionTypes = require( '../constants' ).action,
+	useFakeDom = require( 'test/helpers/use-fake-dom' );
 
 describe( 'team-store', function() {
 
+	useFakeDom();
```

Comment:
This is a really good question :)
I should check if we can remove it completely.